src/main/resources/com/google/api/codegen/php/sample.snip

```diff
@@ -125,6 +125,26 @@
   ?>
 @end
 
+@private nextPageToken(class)
+  ${@class.responseVarName}->get{@class.pageStreaming.nextPageTokenName}()
+@end
+
+@private updatePageToken(class)
+  @if class.pageStreaming.isResourceSetterInRequestBody
+    ${@class.requestBodyVarName}->set{@class.pageStreaming.pageTokenName}({@nextPageToken(class)});
```

Comment:
Would also generally recommend not putting semicolons in the sub snippets and instead appending them to the actual call for consistency

```
...
updatePageToken(class);
```